src/realmGeol.ttl

```diff
@@ -57,307 +288,1624 @@ sorelsp:hasUpperBoundary rdf:type owl:ObjectProperty .
 
 
 ###  http://sweetontology.net/realmGeol/Lopolith
 soreag:Lopolith rdf:type owl:Class ;
-              rdfs:subClassOf soreag:PlutonicStructure ;
-              rdfs:label "lopolith"@en .
+                rdfs:subClassOf soreag:PlutonicStructure ;
+                rdfs:label "lopolith"@en .
 
 
 ###  http://sweetontology.net/realmGeol/LowerMantle
 soreag:LowerMantle rdf:type owl:Class ;
-                 rdfs:subClassOf soreag:Mantle ;
-                 rdfs:label "lower mantle"@en .
+                   rdfs:subClassOf soreag:Mantle ;
+                   rdfs:label "lower mantle"@en .
+
+
+###  http://sweetontology.net/realmGeol/MagnetostratigraphicPolaritySubzone
+soreag:MagnetostratigraphicPolaritySubzone rdf:type owl:Class ;
+                                           rdfs:subClassOf soreag:MagnetostratigraphicUnit ;
+                                           dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                                           dcterms:source <http://www.stratigraphy.org/index.php/ics-stratigraphicguide> ,
+                                                          <https://doi.org/10.1130/9780813774022> ;
+                                           skos:definition "Polarity Zones may be subdivided into Polarity Suzones. The corresponding interval of geological time during which the rocks in the Polarity Subzone were deposited is referred to as its associated Polarity Subchron. (Salvador, 2013; ICS, Online, December, 2019)"@en ;
+                                           skos:prefLabel "magnetostratigraphic polarity subzone"@en .
+
+
+###  http://sweetontology.net/realmGeol/MagnetostratigraphicPolaritySuperzone
+soreag:MagnetostratigraphicPolaritySuperzone rdf:type owl:Class ;
+                                             rdfs:subClassOf soreag:MagnetostratigraphicUnit ;
+                                             dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                                             dcterms:source <http://pid.geoscience.gov.au/dataset/ga/70149> ,
+                                                            <http://www.stratigraphy.org/index.php/ics-stratigraphicguide> ,
+                                                            <https://doi.org/10.1130/9780813774022> ;
+                                             skos:definition "Polarity Zones may be grouped into Polarity Superzones (Hyperzones in Russian literature). The corresponding interval of geological time during which the rocks in the Polarity Superzone were deposited is referred to as its associated Polarity Superchron. As an example of the latter, the mid-Carboniferous – Middle Permian Kiaman Superchron is the longest known period of predominantly reversed polarity, which was terminated by the Illawarra Geomagnetic Polarity Reversal near the base of the Capitanian. (Salvador, 2013; ICS, Online, December, 2019; Gradstein et al., 2012)"@en ;
```

Comment:
skos:definition is not a definition.

Definition: A polarity Superzone is a group of temporally contiguous Polarity Zones, grouped based on ???their remnant magnetization characteristics???.